harvest/trader/trader.py

```diff
@@ -368,42 +322,8 @@ def aggregate_df(self, df, inter):
         else:
             val = 'D'
         df = df.resample(val).agg(op_dict)
-        df.columns = pd.MultiIndex.from_product([[sym], df.columns])
         return df
 
-    def _queue_update(self, df_dict, time):
```

Comment:
What is the equivalent of this deleted code in the Storage class? The Algo class still seems to be making calls to `trader.queue` to get various data so they will need to be updated to point to wherever the queue went.

Reply:
i.e. I might just be missing it but it looks like the data received at `main()` and `main_helper()` in `Trader` is never actually stored anywhere.

Reply:
That data will be saved in the streamer's storage.

Reply:
Hmm, but it doesn't look like it's stored. Both in `_base` and `robinhood` the storage class is instantiated but never used. Also I think it's better to store data in the `Trader` class since other modules need access to the data too, like `Algo`